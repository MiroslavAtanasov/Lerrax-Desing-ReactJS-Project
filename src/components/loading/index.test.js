import React from 'react'
import renderer from 'react-test-renderer'
import Loading from '.'
import TestingEnvironment from '../../test-utils/router'

describe('Loading Component', () => {
    it('should render Loading component', () => {
        const tree = renderer.create(
            <TestingEnvironment value={{
                user: {
                    loggedIn: true,
                    id: '123'
                }
            }}>
                <Loading />
            </TestingEnvironment>
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })
})