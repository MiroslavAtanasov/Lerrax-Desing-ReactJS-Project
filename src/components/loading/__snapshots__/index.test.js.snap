// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component should render Loading component 1`] = `
<div
  className="lds-spinner"
>
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
</div>
`;
